pkgs/tools/misc/rmlint/default.nix

```diff
@@ -18,6 +18,8 @@
 , wrapGAppsHook
 , withGui ? false }:
 
+assert withGui -> !stdenv.isDarwin;
+
```

Comment:
```suggestion
, withGui ? !stdenv.isDarwin }:

```

Reply:
Are you sure? This would change the semantics, as `withGui` defaults to `false`  for `linux` and now it will default to `true` for non-`darwin` `unix`...

Reply:
Maybe that wouldn't be such a good idea.